xbmc/dialogs/GUIDialogBoxBase.h

```diff
@@ -25,6 +25,10 @@
 #include "guilib/GUIDialog.h"
 #include "threads/CriticalSection.h"
 
+#define CONTROL_CANCEL_BUTTON 10
+#define CONTROL_OKAY_BUTTON 11
```

Comment:
There seems to already be a general solution in place for this (`CONTROL_CHOICES_START` and `DIALOG_MAX_CHOICES`) Did you see those?